### PR TITLE
Truncate discord rich presence strings

### PR DESCRIPTION
**Pull request by ElectroJr**
Stops exceptions from being thrown while applying game states.

### CODE REVIEW

File: Robust.Client/Utility/DiscordRichPresence.cs [hunk 1]
```diff
@@ -1,5 +1,8 @@
+using System;
+using System.Text;
 using DiscordRPC;
 using DiscordRPC.Logging;
+using Microsoft.VisualBasic;
```
Comment (moonheart08) [inline Robust.Client/Utility/DiscordRichPresence.cs]: UHHHHHHHHHHH

Comment (ElectroJr) [reply Robust.Client/Utility/DiscordRichPresence.cs]: Ooops. Was a bad namsepace import suggestion when trying to get Encoding.UTF8

File: Robust.Client/Utility/DiscordRichPresence.cs [hunk 2]
```diff
@@ -95,18 +97,58 @@ private void _stop()
...
+            while (encoding.GetByteCount(output) > bytes)
+            {
+                if (value.Length == 0)
+                    return string.Empty;
+
+                value = value.Substring(0, value.Length - 1);
+                output = value + postfix;
+            }
+
+            return output;
```
Comment (metalgearsloth) [inline Robust.Client/Utility/DiscordRichPresence.cs:151]: Yeah I only know string slicing which I don't think does the same.

Comment (ElectroJr) [reply Robust.Client/Utility/DiscordRichPresence.cs:151]: Slicing would be an easy solution if it were limited by character count, Just iteratively removing the last character is pretty shit, but this also shouldn't be getting called much so it should be fine.

**Pull request merged**